app/models/form_profile.rb

```diff
@@ -424,11 +407,6 @@ def phone_object
     home = vet360_contact_info&.home_phone
     return home if home&.area_code && home.phone_number
 
-    if pciu_disabled?
-      # Track precense of home and mobile
-      Rails.logger.info("VAProfile Phone Object: Home? #{home.present?}, Mobile? #{mobile.present?}")
-    end
-
     phone_struct = Struct.new(:area_code, :phone_number)
```

Comment:
Removed pciu_disabled? method and references. Also removed `remove_pciu` flipper